protocol/messenger_backup.go

```diff
@@ -308,6 +327,46 @@ func (m *Messenger) backupCommunities(ctx context.Context, clock uint64) ([]*pro
 	return backupMessages, nil
 }
 
+func (m *Messenger) backupChats(ctx context.Context, clock uint64) []*protobuf.Backup {
+	var oneToOneAndGroupChats []*protobuf.SyncChat
+	m.allChats.Range(func(chatID string, chat *Chat) (shouldContinue bool) {
+		if !chat.Active || (!chat.OneToOne() && !chat.PrivateGroupChat()) {
+			return true
+		}
+		syncChat := protobuf.SyncChat{
+			Id:       chatID,
+			ChatType: uint32(chat.ChatType),
```

Comment:
I guess we should backup props like color, emoji, active, muted, and maybe other relevant props as well?

Reply:
I tried to keep it as light weight as possible.
The color and image props for group chats come from the `MembershipUpdateEvent` (the name is confusing)
I didn't think of syncing the muted property. I guess it makes sense, I'll add it